src/config/wallet.ts

```diff
@@ -138,4 +139,16 @@ export const wallets: WalletConfig<ConnectorNames>[] = [
       return typeof window !== 'undefined' && Boolean(window.ethereum?.isBlocto) ? 0 : 999
     },
   },
+  {
+    title: 'OneKeyWallet',
+    icon: OneKeyWalletIcon,
+    // @ts-ignore
+    installed: typeof window !== 'undefined' && Boolean(window.$onekey?.solana.isOneKey),
```

Comment:
Thanks! You can use `window.$onekey.ethereum.isOneKey` to detect is OneKey in EVM.

Reply:
Updated
Thanks